core/src/main/java/com/linecorp/armeria/common/DefaultResponseHeaders.java

```diff
@@ -17,7 +17,7 @@
 
 import javax.annotation.Nullable;
 
-@SuppressWarnings("checkstyle:EqualsHashCode")
+@SuppressWarnings({ "checkstyle:EqualsHashCode", "EqualsAndHashcode" })
```

Comment:
Could we just leave `"EqualsAndHashcode"`?

Reply:
Good to know that `checkstyle:` prefix is optional. :bow:

Reply:
It seems like Checkstyle only respects the suppressions that start with `checkstyle:`. Reverted.

Reply:
Sorry about that. 😅